project_app/pages/adminpage.tsx

```diff
@@ -0,0 +1,35 @@
+import { useContext, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { GlobalStateContext } from './_app';
+
+
+export default function admin() {
+    const router = useRouter();
+    const { token } = useContext(GlobalStateContext);
+
+    useEffect(() => {
+        if (token == null) {
+            router.push('/admin');
+        } else {
+
+        }

```

Comment:
whats in this else?

Reply:
Actually nothing hehe, will be removing it.